### DISCUSSION TITLE
Deprecated

### DISCUSSION THREAD

**Discussion by ThijsZijdel** [fosslife/devtools-x#89] [Polls]

**Comment by MKRhere** [comment]
Use me as a "Mantine looks better than antd" button:

**Comment by Sparkenstein** [comment]
switching library is not a bad idea tbh, but IMO switching to antd is. When I tried antd a few years ago, they had really bad community support, even today I see 1.1k open issues :P. I remember they just used to close issues hastily ([my experience](https://github.com/ant-design/ant-design/issues/21224)). I do agree that antd is much more widely used and battle tested probably, but I don't want to suffer through UI hacks because library is not correctly written :P.  (community also seem to not prefer antd for large projects) [footnote]

I also agree than current UI with mantine does not look professional, it should look like a "productivity tool" which I think ant-design will be able to do out of the box. antd has pretty clean default UI. 

I chose mantine only because of it's community support and performance.  I don't care about visual elements much. I am open to explore options, but material and the likes are out of the picture. material UI slows down my m2 macbook it's that bad. 

Icons 100% need a new solution. all the icons should be uniform theme. currently we have mix of material + font-aweosme + what not icons. it does not look good imo. any solutions are accepted here. 

I am also open to hire a designer, fixing UI with mantine won't be an issue IMO, but open to options


 * https://www.reddit.com/r/reactjs/comments/sxwyad/any_reason_why_not_to_use_ant_design_in_2022/
  https://www.reddit.com/r/reactjs/comments/kfnerf/thoughts_on_ant_design_vs_other_design_frameworks/
  https://www.reddit.com/r/reactjs/comments/16gvmaw/do_you_recommend_use_ant_design_pros_and_cons/

**Comment by ThijsZijdel** [comment]


**Reply by Sparkenstein** [reply]
Comparing to the existing UI, I think the color contrast is not right. (I wouldn't change the colors honestly, it was picked from some vscode theme after a lots of consideration and a user survey.)

![image](https://github.com/user-attachments/assets/f1ea709f-645e-424c-9c1f-cef21b5c14d6)

**Reply by Sparkenstein** [reply]
I don't mind it to be configurable, but I don't think people will be changing the app colors on daily basis. 
What is your plan here, asking user to edit/create a json to change colors of the app? like vscode theme file?
